### PR TITLE
Fix preserve views node params

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Fix preserve views node params

**Related issue(s)**
Fixes : https://github.com/kyma-project/luigi/issues/337

### CODE REVIEW

File: core/src/App.html [hunk 1]
```diff
@@ -115,8 +115,10 @@
     if (data.params.preserveView) {
       const pv = component.get().preservedViews;
       const nextPath = buildPath(component, data.params);
+      const pathUrlRaw = component.get().pathUrlRaw;
```
Comment (parostatkiem) [inline core/src/App.html]: The solution works well 👍 
I can see you're calling `component.get()` multiple times. Wouldn't it be better to use 
```javascript
const {pathUrlRaw, preservedViews, currentNode} = component.get();
```
I'm not sure but there might be also a performance advantage for this approach.

Comment (jesusreal) [reply core/src/App.html]: I like this suggestion!

File: core/src/services/routing.js [hunk 2]
```diff
@@ -236,6 +237,7 @@ export const handleRouteChange = async (path, component, node, config) => {
       pathParams,
       isolateView,
       viewGroup,
+      pathUrlRaw,
```
Comment (jesusreal) [inline core/src/services/routing.js]: As the only thing we are doing with `patUrlRaw` is extract the url params in _App.html_, why don't we just save the url params instead?

File: core/src/services/routing.js [hunk 3]
```diff
@@ -148,6 +149,9 @@ export const handleRouteChange = async (
 
     const pathUrlRaw =
       path && path.length ? GenericHelpers.getPathWithoutHash(path) : '';
+    const nodeParamsRaw = pathUrlRaw.split('?')[1]
```
Comment (parostatkiem) [inline core/src/services/routing.js]: I think ` const nodeParamsRaw = pathUrlRaw.split('?')[1];` would do exactly the same.

**Pull request merged**